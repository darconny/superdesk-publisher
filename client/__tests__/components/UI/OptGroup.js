import React from "react";
import OptGroup from "../../../components/UI/OptGroup";
import { render } from "@testing-library/react";

const list = [{ id: 2, name: "el1" }, { id: 3, name: "el2" }];

describe("UI/OptGroup", () => {
  it("renders correctly", () => {
    const { container } = render(
      <OptGroup list={list} valueField="id" nameField="name" label="Custom" />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
