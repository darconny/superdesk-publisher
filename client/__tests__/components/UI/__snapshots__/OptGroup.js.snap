// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/OptGroup renders correctly 1`] = `
<optgroup
  label="Custom"
>
  <option
    value="2"
  >
    el1
  </option>
  <option
    value="3"
  >
    el2
  </option>
</optgroup>
`;
